content/docs/aws/pipeline.md

```diff
@@ -0,0 +1,196 @@
+    name='IRIS Classification pipeline',
+    description='IRIS Classification using LR in SKLEARN'
+)
+def iris_pipeline():
+    iris_task = iris_comp().apply(use_aws_secret('aws-secret', 'AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'))
+
+```
+
+## Support S3 Artifact Store
+
+Kubeflow Pipeline supports different artifact viewers. You can create files in S3 and reference them in output artifacts in your application like beflow.
+
+```python
+metadata = {
+        'outputs' : [
+          {
+              'source': 's3://bucket/kubeflow/README.md',
+              'type': 'markdown',
+          },
+          {
+              'type': 'confusion_matrix',
+              'format': 'csv',
+              'schema': [
+                  {'name': 'target', 'type': 'CATEGORY'},
+                  {'name': 'predicted', 'type': 'CATEGORY'},
+                  {'name': 'count', 'type': 'NUMBER'},
+              ],
+              'source': 's3://bucket/confusion_matrics.csv',
+              # Convert vocab to string because for bealean values we want "True|False" to match csv data.
+              'labels': list(map(str, vocab)),
+          },
+          {
+              'type': 'tensorboard',
+              'source': 's3://bucket/tb-events',
+          }
+        ]
+    }
+```
+
+In order for `ml-pipeline-ui` to read these artifacts,
+
+1. Create a Kubernetes secret `aws-secret` with at least S3 read policies in `kubeflow` namespace.
+
+  ```yaml
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: aws-secret
+    namespace: kubeflow
+  type: Opaque
+  data:
+    AWS_ACCESS_KEY_ID: YOUR_BASE64_ACCESS_KEY
+    AWS_SECRET_ACCESS_KEY: YOUR_BASE64_SECRET_ACCESS
+  ```
+
+  > Note: To get base64 string, run `echo -n $AWS_ACCESS_KEY_ID | base64`
+
+1. Update deployment `ml-pipeline-ui` to use AWS credential environment viariables by running `kubectl edit deployment ml-pipeline-ui -n kubeflow`.
+
+  ```yaml
+  apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: ml-pipeline-ui
+    namespace: kubeflow
+    ...
+  spec:
+    template:
+      spec:
+        containers:
+        - env:
+          - name: AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                key: AWS_ACCESS_KEY_ID
+                name: aws-secret
+          - name: AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: AWS_SECRET_ACCESS_KEY
+                name: aws-secret
+          ....
+          image: gcr.io/ml-pipeline/frontend:0.2.0
+          name: ml-pipeline-ui
+  ```
+
+Here's an example.
+<img src="/docs/images/aws/kfp-viewer-tensorboard.png"
+  alt="Kubeflow Pipeline viewer tensorboard"
+  class="mt-3 mb-3 border border-info rounded">
+
+## Support Tensorboard in Kubeflow Pipeline
+
+ [Tensorboard](/docs/pipelines/sdk/output-viewer/#tensorboard) need some extra settings on AWS.
+
+1. Create a Kubernetes secret `aws-secret` S3 read policies in `kubeflow` namespace, check above example.
```

Comment:
policies in `kubeflow` namespace -> policies in the `kubeflow` namespace

End the sentence after 'namespace' and replace 'check above example' with a sentence describing where to find instructions.